shared/desktop/renderer/style.css

```diff
@@ -4,6 +4,7 @@
 @import '../../common-adapters/icon.css';
 @import '../../common-adapters/text.css';
 @import '../../common-adapters/button.css';
+@import '../../common-adapters/choice-list.css';
```

Comment:
so the original reason these were here was because in some cases (like button.tsx) we had a unified implementation so we couldn't just `import './button.css'` in the file (React native packager didn't like that). But, I've since added the capability to do that safely in a cross platform way so
tldr;
we should remove these imports from here and instead just import them in the `.tsx` files instead